core/loadpoint.go

```diff
@@ -641,7 +641,7 @@ func (lp *Loadpoint) syncCharger() error {
 	}()
 
 	// status in sync
-	if enabled == lp.enabled {
+	if enabled && lp.enabled {
```

Comment:
Wenn in sync möchte ich hier immer mit nil raus. Das && macht das kaputt- wofür brauchen wir das?

Reply:
Damit zumindest noch der Check in L674 

`	if !enabled && lp.charging() {`

erreicht werden kann, falls beide (Charger & LP) in sync sind.